stdlib/public/Darwin/Foundation/NSError.swift

```diff
@@ -433,6 +433,12 @@ public protocol _BridgedStoredNSError :
   init(_nsError error: NSError)
 }
 
+public extension _BridgedStoredNSError {
```

Comment:
Mind moving the conformance here?

Reply:
I cannot due to compiler error, please check https://github.com/apple/swift-corelibs-foundation/pull/2140#discussion_r276886110